### DISCUSSION TITLE
IntelliSense not working for .svelte files on existing project, but not on new projects.

### DISCUSSION THREAD

**Discussion by Trystan-SA** [sveltejs/kit#11080] [Q&A] [answered]
### Describe the bug

For some reason, my IntelliSense stopped working for all .svelte files (routes or components stored inside `lib` folder). It still works for .ts files. The bug occurs when I'm using VScode only.

![Capture d'écran 2023-11-18 144430](https://github.com/sveltejs/kit/assets/43876044/73aad2c4-9b9f-46a7-8f57-1fc523405a98)

If I create a new fresh Sveltekit project it work well again. (screen below is from a fresh install, the one above is from my existing project)

![image](https://github.com/sveltejs/kit/assets/43876044/d693e4f5-7e21-4770-80ff-7ac3ccd2e0db)

I tried to :
- Copy the settings from the new Sveltekit project to the old one (tsconfig, vite.config.ts, svelte.config.js). 
- Clean package-lock.json file, node_modules, build, .svelte-kit folders and updated all dependencies (locals and globals). 
- `npm run check` does not throw any errors.
- Restarted Typescript and Svelte language server and restarted VScode multiples times, nothing changed.
- Updated global packages

Autocompletion work well on the old project when using Jetbrain Webstorm. So definitely a typescript/vscode/sveltekit bug. But I don't know what's the cause of it. Any investigation ideas ? Thanks for your help.

### Reproduction

Can't reproduce the bug in a new environment. And I can't share the broken environment for NDA reasons.

### Logs

```Shell
No errors
```

### System Info

```Shell
System:
    OS: Windows 10 10.0.22621
    CPU: (16) x64 11th Gen Intel(R) Core(TM) i7-11700K @ 3.60GHz
    Memory: 19.49 GB / 31.79 GB
  Binaries:
    Node: 20.9.0 - D:\Node20\node.EXE
    npm: 8.19.1 - D:\Node20\npm.CMD
  Browsers:
    Edge: Chromium (119.0.2151.58)
    Internet Explorer: 11.0.22621.1
  npmPackages:
    @sveltejs/adapter-auto: ^2.1.1 => 2.1.1 
    @sveltejs/kit: ^1.27.6 => 1.27.6 
    svelte: ^4.2.5 => 4.2.5 
    vite: ^4.0.0 => 4.5.0
```

### Severity

annoyance

**Comment by teemingc** [comment]
If you can't provide a reproduction, it's impossible to troubleshoot the issue. I can only suggest incrementally migrating your old project files into the new project to find out what the issue is.

**Comment by jasonlyu123** [comment]
Do you have any VSCode workspace config? It might be a wrong [svelte.language-server.ls-path](https://github.com/sveltejs/language-tools/tree/master/packages/svelte-vscode#sveltelanguage-serverls-path) or [svelte.language-server.runtime](https://github.com/sveltejs/language-tools/tree/master/packages/svelte-vscode#sveltelanguage-serverruntime) config that cause the language server to crash. It may be that the config references non-existent files or a mix-up between these two configs.

**Reply by Trystan-SA** [reply]
I don't use workspace config unfortunately.

**Comment by Trystan-SA** [comment, marked as answer]
I fixed it. When I uninstalled all my packages and restarted the Svelte language server, my intellisense worked again. I then tried to isolate the faulty package and found out it was the fault of the `colors` package. It somehow crashed the Svelte language server. 

The weird thing is that I installed/reinstalled it multiples times (with multiples other packages) in my attempt to find the faulty package. Everytime it was reinstalled, the Svelte language server crashed. But when I finally found it was the fault of this single package `colors`, I reinstalled this one package only, and It started working again.

So now I have the exact same configuration than before (`colors` package is installed with same version) but its working, and I don't know why.

**Reply by jacksonthall22** [reply]
Super weird. I recently changed the setting `terminal.integrated.minimumContrastRatio` 4.5 to 1 before this broke, and changing it back fixed it
